docs/source/tutorial/package_a_project.md

```diff
@@ -111,29 +111,22 @@ The resulting `.whl` packages only contain the Python source code of the Kedro p
 
 The project configuration is provided separately in a `tar.gz` file, also inside the `dist` folder. This compressed version of the config files excludes any files inside the `local` directory.
 
-### Package recipients
+### Run a packaged project
 
-Recipients of the `.whl` file need to have Python and `pip` on their machines, but do not need to have Kedro installed.
-
-A recipient can install the project by calling:
+To run a packaged project it must first be installed. To install the project, run the following command:
 
 ```bash
 pip install <path-to-wheel-file>
 ```
 
-An executable, `spaceflights`, is placed in the `bin` subfolder of the Python install folder, so the project can be run as follows:
+Once your project is installed, it  can be run in two ways:
 
-```bash
-python -m spaceflights
-```
+1. Using the project's CLI.
```

Comment:
In my view, there are two things happening:

- The entry point to the script (which points to `package.__main__`) has the name `<package_name>`. It can be used to run the project.
- The entry point uses Kedro's `run` API, which is decorated using `click`. This allows options and arguments passed to the entry point to be consumed, for instance: `my-project --from-nodes A,B`. In my view, this qualifies as a CLI for the packaged project.

I feel that this 'CLI' approach to running a project is also more intuitive to beginner users than using `python`'s `-m` option. What do you think @stichbury?

> I think we should also explain what does python -m does, under the hood it picks up the package.__main__ which user actually has the ability to change before they package it.

I agree with you that this would be generally useful information, but I personally feel that this is too involved for our tutorial. A minority of advanced users may want to modify this behaviour and my view is that they are likely to have sufficient knowledge of Python packaging to achieve this independently. I would avoid overwhelming beginner and intermediate users with information.